src/dcos_e2e/backends/_docker/__init__.py

```diff
@@ -96,6 +96,124 @@ def _get_fallback_storage_driver() -> DockerStorageDriver:
         return DockerStorageDriver.AUFS
 
 
+class DockerLoopbackVolume():
+    """
+    A loopback device sidecar, created in a Docker container.
+    """
+
+    def __init__(
+        self,
+        size: int,
+        labels: Optional[Dict[str, str]] = None,
+    ) -> None:
+        """
+        Create a loopback device pointing to a block device running
+        in a container. This can be used to provide an unformatted
+        disk to a cluster.
+
+        Args:
+            size: Size of the block device in Megabytes.
+            labels: Docker labels to add to the container.
+        """
+        self.size = size
+        docker_container_labels = labels or {}
+
+        self.container, self.path = \
+            DockerLoopbackVolume.create(size, docker_container_labels)
+
+    @staticmethod
+    def create(
+        size: int,
+        labels: Optional[Dict[str, str]],
+    ) -> Tuple[docker.models.containers.Container, str]:
+        """
+        Create a loopback device pointing to a block device running
+        in a container.
+
+        Args:
+            size: Size of the block device in Megabytes.
+            labels: Docker labels to add to the container.
+        Returns:
+            A tuple containing the container and the path of
+            the loopback device.
+        """
+        client = docker.from_env(version='auto')
+
+        container = client.containers.create(
```

Comment:
If `centos:latest` is not already on the host, this `create` would fail; We need to pull it here to make sure.

Reply:
I think `create` does create and pull - but please check.